plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestStandardColumnMappings.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import org.h2.tools.SimpleResultSet;
+import org.testng.annotations.Test;
+
+import java.sql.SQLException;
+import java.sql.Time;
+import java.sql.Types;
+
+import static io.trino.plugin.jdbc.StandardColumnMappings.timeColumnMappingUsingSqlTime;
+import static org.testng.Assert.assertTrue;
+
+public class TestStandardColumnMappings
+{
+    @Test
+    public void testTimeColumnMappingUsingSqlTimeWithNegativeTime()
+            throws SQLException
+    {
+        LongReadFunction readFunction = (LongReadFunction) timeColumnMappingUsingSqlTime().getReadFunction();
+
+        // Create a result set which causes sqlTime.getTime to return a negative value
+        SimpleResultSet rs = new SimpleResultSet();
```

Comment:
The read and write functions are integration glue code and should be tested in an integration manner.
For cases when logic is not trivial, we can test utility methods, however this doesn't make sense when all we test is just a formula.

I guess it is currently not possible to write an integration test because the tests' JVM zone is west of Greenwich, so all legal time values correspond to positive timestamps (millis since epoch) values.

To properly test the thing in an integration manner, we would need a separate test module, or a maven profile, running tests with a different JVM zone. I think it could be considered an overkill just for this fix, and I currently not recall other issues like this one, so I am not convinced this is the way to do. I am OK merging this change without a test.

Reply:
Makes sense, will remove the test.